core/lib/refinery/core/engine.rb

```diff
@@ -79,15 +79,39 @@ def refinery_inclusion!
         ].flatten
       end
 
-      # set the manifests and assets to be precompiled
-      # refinery/icons/* may still be used by plugins
+
+      # set assets to be precompiled
       config.to_prepare do
+
+        # scripts
         Rails.application.config.assets.precompile += %w(
-          refinery/*
+          refinery/*.js
           refinery/icons/*
           modernizr-min.js
           admin.js
         )
+
+        # stylesheets
+        Rails.application.config.assets.precompile += %w(
+          refinery/refinery.css
```

Comment:
Why don't we allow *.css as we do on *.js ?

At this time, i've to add `refinery/*` in my `Rails.application.config.assets.precompile` in assets.rb to be able to use refinery extension who have CSS/JS assets.